CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.16.3 (2021-04-14)
+
+- Fixed the selection box component to have a responsive layout at smaller screen sizes. [#255](https://github.com/blackbaud/skyux-theme/pull/255)
```

Comment:
```suggestion
- Fixed the selection box component to have a responsive layout at smaller screen sizes. [#255](https://github.com/blackbaud/skyux-forms/pull/255)
```
Not sure if change is correct. And, the other link (skyux-theme) is broken.

Reply:
Good catch! Fixed.